CHANGELOG.md

```diff
@@ -34,10 +34,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Dependencies
 
-| Dependency | Old version | New version |
-| ---------- | ----------- | ----------- |
-| `MultiQC`  | 1.25.1      | 1.27        |
-| `MuSE`     |             | 2.1.2       |
+| Dependency                             | Old version | New version |
+| -------------------------------------- | ----------- | ----------- |
+| `bcftools`                             | 1.2         | 1.21        |
```

Comment:
```suggestion
| `bcftools`                             | 1.20         | 1.21        |
```

Reply:
Good catch, I took the `1.2` value from the snapshot.
Somehow the `1.20` got translated as a number to `1.2` and this is what we've been reporting so far

Reply:
Let's report that issue somewhere